automation_tools/__init__.py

```diff
@@ -1322,18 +1322,18 @@ def product_install(distribution, create_vm=False, certificate_url=None,
     if distribution.startswith('satellite6'):
         execute(setup_default_docker, host=host)
         execute(katello_service, 'restart', host=host)
+        if os.environ.get('LIBVIRT_KEY_URL') is not None:
```

Comment:
Can we add a comment explaining the logic here? 3 months from now we want to remember why we have this condition :)